src/main/kotlin/racingcar/view/input/InputConsole.kt

```diff
@@ -12,10 +12,10 @@ class InputConsole : Input {
     override fun finalLab(): Int = inputToCorrectIntValue()
 
     private fun inputToCorrectStringValue(): List<String> {
-        var carNames: List<String>
+        val carNames: MutableList<String> = mutableListOf()
 
         do {
-            carNames = inputNamesOfCars()
+            carNames.addAll(inputNamesOfCars())
         } while (carNames.isEmpty())
 
         return carNames
```

Comment:
```suggestion
        return carNames.toList()
```
방어적 복사를 해서 리턴해주시면 좋을 것 같아요

Reply:
아 그렇군요.. `mutable list` 이기 때문에 더더욱 변경해 반환하는 것이 맞겠네요!